templates/components.html

```diff
@@ -183,7 +175,7 @@ <h6 class="card-subtitle text-muted" align="center">
               <center>
                 <img
                   class="cards_img lazy-load-img"
-                  data-src="static/images/papers/{{openreview.UID}}.png"
+                  data-src="https://iclr.github.io/iclr-images/{{openreview.UID}}.png"
```

Comment:
I guess this is being pulled from upstream too? It's changing our link to the ICLR website. I assume it's not used anywhere, but maybe better to keep this kind of thing as it is?

Reply:
Yes. Pulled from upstream for clean start. Will be updated when new content is available. Ideally, paths that can change based on instances of MiniConf should be coming from a config file or a single source of truth. This would be a good fix and candidate for a PR to the MiniConf repo.